src/ckb/channel.rs

```diff
@@ -2897,6 +2919,87 @@ impl ChannelActorState {
         Ok(())
     }
 
+    fn handle_reestablish_channel_message(
+        &mut self,
+        reestablish_channel: ReestablishChannel,
+        network: &ActorRef<NetworkActorMessage>,
+    ) -> ProcessingChannelResult {
+        match self.state {
+            ChannelState::NegotiatingFunding(_flags) => {
+                // TODO: in current implementation, we don't store the channel when we are in NegotiatingFunding state.
+                // This is an unreachable state for reestablish channel message. we may need to handle this case in the future.
+            }
+            ChannelState::ChannelReady() => {
+                // resend AddTlc, RemoveTlc and CommitmentSigned messages if needed
+                let local_commitment_number = reestablish_channel.local_commitment_number;
+                let mut need_resend_commitment_signed = false;
+                for info in self.tlcs.values() {
+                    if info.is_offered() {
+                        if info.created_at.get_local() > local_commitment_number
```

Comment:
Local/remote are always relative. If we received a `local_commitment_number`, that is there local commitment number. And it should be our remote commitment number.